src/IOUService/web/ymmd.js

```diff
@@ -38,10 +40,35 @@ function load_sent_notes() {
 }
 
 function show_sent_notes(notes) {
-    clear_table();
 
-    var table = document.getElementById("sent_notes_table");
-    $.each(notes, function(index, value) {
+    display_notes_table("sent_notes_table", notes);
+}
+
+function set_up_refresh_button() {
+    $("#refresh_button").click(function () {
+        load_received_notes();
+    });
+}
+
+function load_received_notes() {
```

Comment:
idea: this and the other load method looks pretty similar. We might want to extract the commonality (if the result is clearer)